cmd/clusterctl/client/cluster/workload_cluster.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cluster
+
+import (
+	kc "sigs.k8s.io/cluster-api/util/kubeconfig"
```

Comment:
nit 
can we leave kubeconfig (kc is an uncommon abbreviation)

Reply:
@fabriziopandini Initially I used it like as it is but the ci-make job gives me an error that "kubeconfig already declared through import of package kubeconfig ("sigs.k8s.io/cluster-api/util/kubeconfig) in file sigs.k8s.io/cluster-api/cmd/clusterctl/client/cluster/proxy_test.go"

Reply:
I see. It is conflicting with the function `kubeconfig(currentContext, namespace string)`. In that case I suggest changing this to `utilkubeconfig`. 
I've seen this pattern used elsewhere. E.g. https://github.com/kubernetes-sigs/cluster-api/blob/e020060957ca4cd1e0b8bf2ae02b1c26ed85ffa1/cmd/clusterctl/client/cluster/proxy.go#L27

Another option would be to change the `kubeconfig` function name but that might be a much bigger diff.

@fabriziopandini WDYT?